WISE-Community/WISE-Client
# Manage Students: Show all periods

## Changes
- Show all the periods in Manage Students view when "All Periods" is selected in the period dropdown.

Closes #286.

## Test
Make sure that all Manage Students functions work as before:
- Change period
- Move student to different team within period
- View student info
